### ISSUE TITLE
Change made using Jottacloud web UI not picked up (quickly) by jotta-cli sync

### ISSUE THREAD

**Issue by aerique** [jotta/jotta-cli-issues#134]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):** 0.9.40946

**Description of problem:** Using the Jottacloud web UI I copied a folder (`my-folder`) from `Archive` to `Synced` and it did not show up / get synced to my machine until I did a `touch foo` on the command line in my Jottacloud sync folder. (I waited about 5 minutes before doing creating the `foo` file.)

**Expected:** The `my-folder` to show up in my Jottacloud sync directory on my local machine.

**jotta-cli status (`jotta-cli status`):**
```
> jotta-cli status
------------------------------------------------------------------------------
 Account   : emailadress@example.com on Jottacloud
 Usage     : 1.04TiB / 5.00TiB 
 Device    : desktop@work
------------------------------------------------------------------------------
 Sync      :
------------------------------------------------------------------------------
   Location  : /home/userJottacloud
   Count     : 38167 files 4.06GiB
   Mode      : realtime ( default )
   Updated   : Tue Feb 23 14:09:49
------------------------------------------------------------------------------
 No folders added to backup yet
------------------------------------------------------------------------------
OK
```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
```
pid:2226 2021/02/23 13:18:42 levent Path /foo
pid:2226 2021/02/23 13:18:42 levent Path /foo
pid:2226 2021/02/23 13:18:44 Starting upload of 1 files
pid:2226 2021/02/23 13:18:44 Finished upload of 1 files
pid:2226 2021/02/23 13:18:44 sync.localactions 1 ok
pid:2226 2021/02/23 13:18:44 Starting sync-complete  => 
pid:2226 2021/02/23 13:18:44 revent NEW_UPLOAD path:(/stmmfu5ja3hjfm1tjk2he6ne/Jotta/Sync/foo) mod:(1614082722 => 2021-02-23 12:18:42 +0000 UTC)
pid:2226 2021/02/23 13:18:57 Starting create-local-folder /my-folder
pid:2226 2021/02/23 13:18:57 levent Path /my-folder
```

**Comment by Kimbsen** [comment]
Thanks for the report! 

We've identified why this happens and it will get fixed for the next release.

**Comment by Kimbsen** [comment]
This should be fixed in version 0.9.41047 (unstable)
